confluent-flink-quickstart/confluent-flink-quickstart.py

```diff
@@ -0,0 +1,175 @@
+
+
+def get_cluster_id_for_flink_pool(_candidate_clusters):
+    if _candidate_clusters:
+        cluster_with_topics = {}
+        if debug:
+            print(f'Using databases {_candidate_clusters}')
+        for cluster in _candidate_clusters:
+            _cluster_id = cluster['id']
+            topics_json = cli(["confluent", "kafka", "topic", "list", "--cluster", _cluster_id, '-o', 'json'])
+            topics = []
+            for topic in topics_json:
+                topics.append(topic['name'])
+            cluster_with_topics[_cluster_id] = (Cluster(_cluster_id, cluster['name'], topics))
+            if debug:
+                print(f'Found topics {topics} for cluster {_cluster_id}')
+
+        print("Found the following databases with tables")
+        print(table_format.format("CLUSTER ID", "CLUSTER NAME", "TOPICS"))
+        for cluster in cluster_with_topics.values():
+            print(table_format.format(cluster.cid, cluster.name, str(cluster.topics)))
+
+        while True:
+            choice = input("Enter a Kafka cluster ID to use or 'create' to use a new one for Flink > ")
+            if choice:
+                if choice.lower() == 'create':
+                    _cluster_id = create_cluster_with_schema_registry()
+                    break
+                if choice in cluster_with_topics:
+                    _cluster_id = choice
+                    break
+
+            print(f'{choice} is not valid')
+
+    else:
+        _cluster_id = create_cluster_with_schema_registry()
+
+    return _cluster_id
+
+
+def process_cluster_list(existing_clusters, curr_flink_region):
+    if not existing_clusters:
+        print(f'No existing database found, will create one in {curr_flink_region}')
+    else:
+        clusters = []
+        for existing_cluster in existing_clusters:
+            if existing_cluster['region'] == curr_flink_region:
+                kafka_cluster = existing_cluster['name']
+                clusters.append(existing_cluster)
+                if debug:
+                    print(f'Found a Kafka cluster, {kafka_cluster} with region {curr_flink_region}')
+
+        if not clusters and debug:
+            print(f'No exising database found in region {curr_flink_region}, will create one')
+        return clusters
+
+
+usage_message = '''confluent flink quickstart [-h] --name NAME [--units NUM-UNITS] [--env ENV] [--region REGION] '''
+
+parser = argparse.ArgumentParser(description='Creates a Flink compute pool.\n'
+                                             'Looks for exising Kafka clusters '
+                                             'and prompt the user to select one as a database for the Flink pool. \n'
+                                             'Creating one is an option as well.\n'
+                                             'If there are no existing clusters, the plugin will create one.\n'
+                                             'Then it starts a Flink SQL cli session'
```

Comment:
to match [docs](https://docs.confluent.io/cloud/current/flink/get-started/quick-start-shell.html#step-2-start-the-flink-sql-shell)

```suggestion
                                             'Then it starts a Flink SQL shell'
```